src/serac/physics/operators/odes.cpp

```diff
@@ -25,27 +25,91 @@ void SecondOrderODE::SetTimestepper(const serac::TimestepMethod timestepper)
 {
   switch (timestepper) {
     case serac::TimestepMethod::HHTAlpha:
-      ode_solver_ = std::make_unique<mfem::HHTAlphaSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::HHTAlphaSolver>();
       break;
     case serac::TimestepMethod::WBZAlpha:
-      ode_solver_ = std::make_unique<mfem::WBZAlphaSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::WBZAlphaSolver>();
       break;
     case serac::TimestepMethod::AverageAcceleration:
-      ode_solver_ = std::make_unique<mfem::AverageAccelerationSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::AverageAccelerationSolver>();
       break;
     case serac::TimestepMethod::LinearAcceleration:
-      ode_solver_ = std::make_unique<mfem::LinearAccelerationSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::LinearAccelerationSolver>();
       break;
     case serac::TimestepMethod::CentralDifference:
-      ode_solver_ = std::make_unique<mfem::CentralDifferenceSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::CentralDifferenceSolver>();
       break;
     case serac::TimestepMethod::FoxGoodwin:
-      ode_solver_ = std::make_unique<mfem::FoxGoodwinSolver>();
+      second_order_ode_solver_ = std::make_unique<mfem::FoxGoodwinSolver>();
+      break;
+    case serac::TimestepMethod::NewmarkBeta:
+      second_order_ode_solver_ = std::make_unique<mfem::NewmarkSolver>();
+      break;
+    case serac::TimestepMethod::BackwardEuler:
+      first_order_system_ode_solver_ = std::make_unique<mfem::BackwardEulerSolver>();
       break;
     default:
       SLIC_ERROR("Timestep method was not a supported second-order ODE method");
   }
-  ode_solver_->Init(*this);
+
+  if (second_order_ode_solver_) {
+    second_order_ode_solver_->Init(*this);
+  } else if (first_order_system_ode_solver_) {
+    // we need to adjust the width of this operator
+    width *= 2;
+    first_order_system_ode_solver_->Init(*this);
+  }
```

Comment:
A else clause slic error would prevent this from ever failing to define an ODE solver.

Reply:
I've added the SLIC_ERROR because I think it might help catch any developer implementation errors. It shouldn't be possible for users to muck with this.